src/platform/windows/startup-issues.md

```diff
@@ -65,21 +65,20 @@ If the update process leaves you with multiple Anki installs (such as within
 `C:\Program Files\Anki` and `C:\Program Files (x86)\Anki`), they may be left in a
 non-working state, and Anki may refuse to start without showing an error message.
 
-Try uninstalling all copies - you may be able to do so with the Windows `Apps & features` settings menu, or by running `uninstall.exe` in each Anki program
+Try uninstalling all copies - from Windows Settings > Apps & features (or Apps > Installed apps), or by running `uninstall.exe` in each Anki program
 folder. Afterward, install Anki again.
 
 ## Debugging
 
 Starting Anki from a terminal may reveal a bit more information about some
 errors. After installing the latest Anki version and ensuring all Windows
-updates are installed, instead of running Anki directly, use Start>Run
-and type cmd.exe. When a console window appears, type
+updates are installed, instead of running Anki directly, press the <kbd>Windows</kbd> key (or open the Start menu), type `cmd`, and launch Command Prompt. When the terminal window opens, paste (or type) the following command, and press <kbd>Enter</kbd>. (The path will be different if Anki is installed in a location that is not the default.)
```

Comment:
```suggestion
updates are installed, instead of running Anki directly, press the <kbd>Windows</kbd> key (or open the "Start" menu), type `cmd`, and launch Command Prompt. When the terminal window opens, input the following command, and press <kbd>Enter</kbd>. (The path will be different if Anki is installed in a location that is not the default.)
```

Reply:
"Input" makes it sound too technical. I'm trying to beginner-proof this description.

Reply:
If you wish you can say enter info. Paste is decent too. Type is a bit non-inclusive.

Reply:
I'll just go with "paste" -- the block has a copy button, and we'll avoid more errors that way.